src/configs.py

```diff
@@ -49,7 +49,7 @@
 
 # we_chat config
 
-GRANT_TYPE = 'client_credentials'
+GRANT_TYPE = 'client_credential'

```

Comment:
I am very sorry for my carelessness. Shame on me .

Reply:
It doesn't matter.